tests/test_uroot.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# Clone repo
+git clone --branch "${VERIFICATION_TEST_UROOT_VERSION}" --depth 1 https://github.com/u-root/u-root.git
+cd u-root
+
+# Make
+go build
+./u-root \
+	-defaultsh gosh \
```

Comment:
Is this actually still needed? Not sure when they made gosh the default upstream.

Reply:
I don't know, I just blindly copied stuff from CodingVoid.